src/components/File/File.test.jsx

```diff
@@ -88,7 +88,9 @@ describe("<File />", () => {
       expect(description.innerHTML).toBe("2 files selected");
     });
 
-    it("should allow resetting if embedded in a form", async () => {
+    // This test is flaky and unreliable. We should manually verify the expected behavior when
+    // making changes or updating dependencies
+    it.skip("should allow resetting if embedded in a form", async () => {
```

Comment:
@pcberg @johglove, I wanted to double check that y'all are ok with this change. It narrowly impacts coverage, but the test failing only occasionally impacts development and DX and this branch can be manually tested during dependency updates.

Reply:
No objections from me.